bin/gpm

```diff
@@ -53,6 +54,17 @@ if (destOpt.value) {
     destOpt.value = Path.resolve(destOpt.value.replace("~", Ul.home()));
 }
 
+if (inputOpt.value === ".") {
+    // Load and parse the package.json file
+    var packageJsonPath = Path.join(process.cwd(), "package.json")
+      , manifest = JSON.parse(Fs.readFileSync(packageJsonPath));
+
+    // Merge all dependencies on a single Array
+    var dependencies = Object.keys(manifest.dependencies)
+      , devDependencies = Object.keys(manifest.devDependencies);
+    inputOpt.value = dependencies.concat(devDependencies);
```

Comment:
Is it possible to add a new method for this? Something like `Gpm.prototype.getDependencies = ...`, and considering the `is_dev` field (`self.is_dev`).